### DISCUSSION TITLE
Crazyflie with AI Deck, Multiranger Deck and Markers

### DISCUSSION THREAD

**Discussion by mmcza** [bitcraze/discussions#1645] [Q&A] [answered]
Hi,

For a project at university, I want to use Crazyflie with AI and Multiranger Decks, but I don't want to implement a localization algorithm, but instead use groundtruth from OptiTrack. Here are my questions:
- would the drone be able to lift with the weight of those 3 decks? If not, is there some good way to place some markers on the drone instead of using the official deck?
- what would be the best way to place the Decks? I saw a paper, where they put the AI and Multiranger Decks on top of the drone, but they didn't have the markers, and I would like to do it in a way that the markers don't block the distance sensors.

Thanks in advance

**Comment by Bananazooka** [comment, marked as answer]
Hi, 
It's possible to have 3 decks! You could have the AI deck on the bottom, and the Multi Ranger on top, with the Motion Capture deck in the middle. The Multiranger has a sensor that looks up so you shouldn't put anything on top of it. You can check the compatibility matrix [here](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck---deck) when in doubt.
In your case though, you can have the reflective markers attached to the Crazyflie without the Motion Capture Marker deck which would avoid the congestion.

**Reply by mmcza** [reply]
Thank you for the response, and sending the link to the compatibility matrix, and suggestion how to attach everything to the drone.
